Resources/Prototypes/_Goobstation/Entities/Structures/Doors/Airlocks/airlocks.yml

```
@@ -25,4 +28,33 @@
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.49,-0.49,0.49,0.49"
+        density: 100
+        mask:
+          - FullTileMask
+        layer:
+          - GlassLayer
+  - type: RadiationBlocker
+    resistance: 20
+  - type: Door
+    occludes: false 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)